app/policies/user_policy.rb

```diff
@@ -0,0 +1,30 @@
+class UserPolicy
+  attr_reader :current_user, :user
+  
+  def initialize(current_user, user)
+    @current_user = current_user
+    @user = user
+  end
+  
+  def discover?
+    true
+  end
+  
+  def feed?
+    true
+  end
+  
+  def liked?
+    true
+  end
```

Comment:
In this lesson, we're not implementing Likes, so this isn't needed.

Reply:
+1